images/public-log-vendor-matcher/README.md

```diff
@@ -0,0 +1,44 @@
+# Public log vendor matcher
+
+A Postgres-based k8s-infra data pipeline that produces BigQuery tables for reviewing Kubernetes Public artifact traffic in DataStudio.
+
+## Environment variables
+
+| Name                             | Default                                                | Description                                                                           |
+| -------------------------------- | ------------------------------------------------------ | ------------------------------------------------------------------------------------- |
+| `TZ`                             | ``                                                     | Container time zone                                                                   |
+| `GOOGLE_APPLICATION_CREDENTIALS` | ``                                                     | The path to the GCP service account json key                                          |
+| `GCP_PROJECT`                    | `k8s-infra-ii-sandbox`                                 | The project to target                                                                 |
+| `GCP_SERVICEACCOUNT`             | `asn-etl@k8s-infra-ii-sandbox.iam.gserviceaccount.com` | The GCP service account name                                                          |
+| `GCP_BIGQUERY_DATASET`           | `etl_script_generated_set`                             | The dataset and basename to write to (appends date)                                   |
+| `NO_PROMOTE`                     | ``                                                     | Disable the promotion of `${GCP_BIGQUERY_DATASET}_${DATE}` to ${GCP_BIGQUERY_DATASET} |
+| `ASN_DATA_PIPELINE_RETAIN`       | ``                                                     | Keeps Postgres running after the job has completed                                    |
+
+## Running the Pipeline Manually
+
+Generate a key file for ServiceAccount auth
+
+```
+gcloud iam service-accounts keys create /tmp/asn-etl-pipeline-gcp-sa.json --iam-account=asn-etl@k8s-infra-ii-sandbox.iam.gserviceaccount.com
+```
```

Comment:
We need to not do this.  I do not want these keys laying around.

Realistically if you're running this as a developer, it should be running with _your_ credentials, not as a service account.

As a super last resort if you truly need to repro acting as the service account, we could investigate using `--impersonate-service-account` (ref: https://cloud.google.com/sdk/gcloud/reference/#--impersonate-service-account)